cmd/influx_tsm/stats/stats.go

```diff
@@ -18,30 +18,37 @@ type Stats struct {
 	TotalTime       time.Duration
 }
 
+// AddPointsRead adds read points
```

Comment:
Thank you, @jsternberg, for the example.  Let me know if you have other suggestions. I can do more on the weekend and will follow the same format/guidelines.